packages/doenetml-worker-rust/lib-doenetml-core/src/core/components/common.rs

```diff
@@ -118,14 +124,11 @@ pub trait ComponentNode: ComponentState {
     /// Get the index of the parent node
     fn get_parent(&self) -> Option<ComponentIdx>;
     /// Get the vector containing the indices of all child component nodes and the literal string children.
-    fn get_children(&self) -> &Vec<ComponentPointerTextOrMacro>;
+    fn get_children(&self) -> &Vec<UntaggedContent>;
     /// Set the vector containing the indices of all child component nodes and the literal string children.
-    fn set_children(&mut self, children: Vec<ComponentPointerTextOrMacro>);
+    fn set_children(&mut self, children: Vec<UntaggedContent>);
     /// Replace with new values the vector containing the indices of all child component nodes and the literal string children.
-    fn replace_children(
-        &mut self,
-        new_children: Vec<ComponentPointerTextOrMacro>,
-    ) -> Vec<ComponentPointerTextOrMacro>;
+    fn replace_children(&mut self, new_children: Vec<UntaggedContent>) -> Vec<UntaggedContent>;
```

Comment:
Could delete `replace_children` from trait as it is no longer needed with the `ComponentBuilder`.